### DISCUSSION TITLE
microweber two factor authentication

### DISCUSSION THREAD

**Discussion by bertiebw** [microweber/microweber#1027] [Q&A]
Good morning,
I am looking for information on how to implement/enable two factor authentication on microweber.
Searching around did not return any result. I am not sure if I missed something.
Would you somebody be able to help please?
Thanks.
B

**Comment by NoSpamMaam** [comment]
Thought TFA integrated by default... new to the platform myself and haven't given it a spin just thought I saw it in a video somewhere...but maybe it was a preinstalled module?

**Comment by peter-mw** [comment]
Hi , 2FA has been integrated in version 2.0

On in Admin->Users->Profile

![image](https://github.com/microweber/microweber/assets/5698247/4cd354fa-7531-45d0-bf06-0abe4d4dfdf7)
